sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
if the db table is the issue i can rewrite it to "store" them in files.
i think that is not that hard to add new sensors to it, but at this time i have acces only to these 2 hardware types

the "component" table should be altered for this purpose, but if there is no issue with it i can change the code to use it